tests/analysis/xas/test_spectrum.py

```diff
@@ -38,22 +38,22 @@ def setUp(self):
         self.site2_xanes = XAS.from_dict(site2_xanes_dict)
 
     def test_e0(self):
-        assert approx(self.k_xanes.e0) == 7728.565
```

Comment:
I believe `a == pytest.approx(ref_val)` is the [recommended usage](https://docs.pytest.org/en/stable/reference/reference.html) plus it's slightly more readable. However do note `approx` is not symmetric:
> a == pytest.approx(b, rel=1e-6, abs=1e-12): True if the relative tolerance is met w.r.t. b or if the absolute tolerance is met. Because the relative tolerance is only calculated w.r.t. b, this test is asymmetric and you can think of b as the reference value. In the special case that you explicitly specify an absolute tolerance but not a relative tolerance, only the absolute tolerance is considered.